FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -387,14 +387,33 @@ open scoped TensorProduct -- ⊗ notation for tensor product
 variable [Algebra K (ProdAdicCompletions B L)]
   [IsScalarTower K L (ProdAdicCompletions B L)]
 
+
 noncomputable def ProdAdicCompletions.baseChange :
     ProdAdicCompletions A K →ₐ[K] ProdAdicCompletions B L where
   toFun kv w := (adicCompletionComapAlgHom A K L B _ w rfl (kv (comap A w)))
-  map_one' := sorry -- #232 is this and the next few sorries. There is probably a cleverer way to do this.
-  map_mul' := sorry
-  map_zero' := sorry
-  map_add' := sorry
-  commutes' := sorry
+  map_one' := by
+    simp only
```

Comment:
```suggestion
    dsimp only
```
It's even quicker because it applies even less stuff, but still does what you need. And below.